tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
+ * Usage: php loris_db_upgrade.php [-D]
+ *
+ * PHP Version 7
+ *
+ * @category Install
+ * @package  Loris
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ * @package behavioural
+ */
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . "/generic_includes.php";
+ set_include_path(get_include_path().":../project/libraries:../php/libraries:");
+require_once "../php/libraries/Database.class.inc";
+require_once "../php/libraries/NDB_Config.class.inc";
+$path    = __DIR__ . "/../SQL/Release_patches";
+
+$files = array_diff(scandir($path), array('.', '..'));
+
+echo "This script will upgrade the loris database to the lastest.\n";
+echo "Backing Up the Database Before Upgrading the Database.\n";
+echo "User must be granted all necessary privileges (ALTER TABLE, INSERT, UPDATE, DELETE ...) on Loris database.\n";
```

Comment:
The script use the regular user from config.xml which do not have permission to modify the schema.
Instead to echoing this file, should manually open a connection to the DB using quatuser.
see https://github.com/aces/Loris/pull/5260

Reply:
the quatuser was for the old DQT and shouldn't even have delete permission.  Would be good to rename/fix the credential...